csrankings-a.csv

```diff
@@ -393,7 +393,8 @@ Aisling Kelliher,Virginia Tech,http://people.cs.vt.edu/~aislingk,71RtIxoAAAAJ
 Aiswarya Cyriac,CMI,http://www.cmi.ac.in/~aiswarya,NLwqvCwAAAAJ
 Aivar Annamaa,University of Tartu,http://www.ut.ee/en/kontakt/arvutiteaduse-instituut,sQ5LoEcAAAAJ
 Aixia Jia,Peking University,http://59.108.48.38:8080/ICSTWIPWeb/jiaaixia.php,NOSCHOLARPAGE
-Aixin Sun,Nanyang Technological University,https://www3.ntu.edu.sg/home/axsun,wyKGVKUAAAAJ
+Aixin Sun,Nanyang Technological University,https://www.ntu.edu.sg/home/axsun,wyKGVKUAAAAJ
+Aixin Zhang,Shanghai Jiao Tong University,https://infosec.sjtu.edu.cn/DirectoryDetail.aspx?id=148,NOSCHOLARPAGE

```

Comment:
Listed as "associate researcher". Only tenure-track faculty are eligible for inclusion.